lib/CloudinaryPlugin.js

```diff
@@ -98,20 +97,16 @@ class CloudinaryPlugin {
       }
     ];
 
-    this.config = givenConfig;
+    this.config = config;
 
     const { NODE_ENV } = process.env;
-    const missingCredMessage = 'Cloudinary Plugin cannot be used without the ApiKey, the ApiSecret and the CloudName of your Cloudinary Account';
-
     const { cloudinaryCloudName, cloudinaryApiKey, cloudinaryApiSecret } = this.config;
 
-    //Check if credentials are missing 
     if (!cloudinaryApiKey || !cloudinaryApiSecret || !cloudinaryCloudName) {
       if (NODE_ENV === 'production') {
-        throw new this.context.errors.InternalError(missingCredMessage);
+        throw new this.context.errors.InternalError('Cloudinary Plugin cannot be used without the ApiKey, the ApiSecret and the CloudName of your Cloudinary Account');
       } else {
-        //TODO Kuzzle Logger
-        this.context.log.error(missingCredMessage);
+        this.context.log.error('Cloudinary Plugin cannot be used without the ApiKey, the ApiSecret and the CloudName of your Cloudinary Account');
```

Comment:
(nitpicking) wouldn't be a bad thing to tell what will happen outside of dev mode

```suggestion
        this.context.log.error('Cloudinary Plugin cannot be used without the ApiKey, the ApiSecret and the CloudName of your Cloudinary Account.\nUsing this plugin outside of development without those keys will prevent Kuzzle from starting.');
```